### PR TITLE
Make atom feeds great again

### LINKED ISSUE #61: Display an atom feed icon on the tracker
Currently users have to dive in the code and discover that an atom feed exists. Add an icon to the main page which redirects to the atom feed.

### PR DESCRIPTION
**Pull request by ysf**
This uses feedgen to provide atom feeds after werkzeugs atom_feed got deprecated. Fixed #61 on the same run.

### CODE REVIEW

**Comment by jelly** [comment]
This required feedgen to be packaged in Arch Linux and requires it to be added along with ```python-pytz``` to the packages listed in ```.travis.yml```.

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,7 @@
 __pycache__/
+asset/
```
Comment (jelly) [inline .gitignore:2]: Hmm what's this directory? :thinking:

Comment (ysf) [reply .gitignore:2]: not needed here.

File: requirements.txt [hunk 2]
```diff
@@ -8,3 +8,5 @@ requests
 scrypt
 pyalpm
 SQLAlchemy-Continuum
+feedgen
+pytz
```
Comment (anthraxx) [inline requirements.txt:12]: Right now we also duplicate the required packages in the README.md, that will need adjustments as well

File: test/test_advisory.py [hunk 3]
```diff
@@ -342,21 +342,17 @@ def test_advisory_cve_listing_sorted_numerically(db, client):
 
 def test_advisory_atom_no_data(db, client):
     resp = client.get(url_for('tracker.advisory_atom'), follow_redirects=True)
-    assert 404 == resp.status_code
-    # TODO: re-enable feed test
-    # data = resp.data.decode()
-    # assert DEFAULT_ADVISORY_ID not in data
+    data = resp.data.decode()
+    assert DEFAULT_ADVISORY_ID not in data
```
Comment (anthraxx) [inline test/test_advisory.py:346]: i think we may want to test for a valid `200` return code as well

File: test/test_advisory.py [hunk 4]
```diff
@@ -342,21 +342,17 @@ def test_advisory_cve_listing_sorted_numerically(db, client):
...
 @create_group(id=DEFAULT_GROUP_ID, packages=['foo'], affected='1.2.3-3', fixed='1.2.3-4')
 @create_advisory(id=DEFAULT_ADVISORY_ID, group_package_id=DEFAULT_GROUP_ID, advisory_type=issue_types[1], reference='https://security.archlinux.org', publication=Publication.published)
 def test_advisory_atom(db, client):
     resp = client.get(url_for('tracker.advisory_atom'), follow_redirects=True)
-    assert 404 == resp.status_code
-    # TODO: re-enable feed test
-    # data = resp.data.decode()
-    # assert DEFAULT_ADVISORY_ID in data
+    data = resp.data.decode()
+    assert DEFAULT_ADVISORY_ID in data
```
Comment (anthraxx) [inline test/test_advisory.py:355]: i think we may want to test for a valid `200` return code as well

File: tracker/templates/advisories.html [hunk 5]
```diff
@@ -2,7 +2,7 @@
 {%- from "_formhelpers.html" import nullable_value, colorize_severity, colorize_status, bug_ticket -%}
 {% block content %}
 			<h1>Advisories
-				<a href="{{ url_for('tracker.advisory_atom') }}">feed</a>
+                <a href="{{ url_for('tracker.advisory_atom') }}"><img src="/static/feed.svg" alt="rss feed" height="18" width="18" /></a>
```
Comment (anthraxx) [inline tracker/templates/advisories.html:5]: This line changes tabs to spaces, the html part is tab indented.

File: tracker/view/advisory.py [hunk 6]
```diff
@@ -1,7 +1,10 @@
 from collections import OrderedDict
 from re import match
 
+import pytz
```
Comment (anthraxx) [inline tracker/view/advisory.py:4]: mostly we try to just import what we need, can we changed this to only import `UTC`?

**Comment by wvffle** [comment]
What's the status with on this PR?

**Review by anthraxx — APPROVED** [review]
lgtm

**Comment by anthraxx** [comment]
applied in 0776f8192816d15b438fbdfd8565cc5ff68ab125
thanks a lot for your contribution

**Pull request closed without merging**